src/rgw/rgw_common.cc

```diff
@@ -73,8 +73,8 @@ rgw_err()
 }
 
 rgw_err::
-rgw_err(int http, const std::string& s3)
-    : http_ret(http), ret(0), s3_code(s3)
+rgw_err(int http, const std::string& error)
+    : http_ret(http), ret(0), err_code(error)
```

Comment:
Maybe something like that?
```
rgw_err(int http_ret, const std::string& err_code)
   : http_ret(http_ret), ret(0), err_code(err_code)
```
I mean naming the ctor's parameters exactly as the corresponding class member variables.